docs/index.bs

```diff
@@ -3608,6 +3732,29 @@ spec: webidl; urlPrefix: https://heycam.github.io/webidl/
           <li>Set <var>registration</var> to the result of running <a href="#scope-match-algorithm">Match Service Worker Registration</a> algorithm passing <var>request</var>'s <a for="request">url</a> as the argument.</li>
           <li>If <var>registration</var> is null or <var>registration</var>'s <a href="#dfn-active-worker">active worker</a> is null, return null.</li>
           <li>Set <var>client</var>'s <a href="#dfn-service-worker-client-active-worker">active worker</a> to <var>registration</var>'s <a href="#dfn-active-worker">active worker</a>.</li>
+          <li>If <var>request</var> is a <a>navigation request</a> and <var>registration</var>'s <a>navigation preload enabled flag</a> is set, and <var>request</var>'s <a>method</a> is `<code>GET</code>`, then:
```

Comment:
@annevk this is the section I'm least confident in - it's fetch for the preload. I don't think redirects are a problem since the mode will be `manual`. When the redirect is sent back to the navigate algorithm it'll be associated with the *original* request, which won't have the header.

Having said that, that may be unexpected for same-origin redirects.